azure/pipelines/templates/code-coverage.yml

```diff
@@ -0,0 +1,67 @@
+parameters:
+- name: vmImage
+  displayName: 'Agent'
+  type: string
+  default: 'windows-latest'
+
+jobs:
+- job: code_coverage
+  displayName: Code Coverage
+  pool:
+    vmImage: ${{ parameters.vmImage }}
+
+  variables:
+    Solution_File: $(System.DefaultWorkingDirectory)/source/F0.Minesweeper.sln
+    NuGet_Configuration_File: $(System.DefaultWorkingDirectory)/nuget.config
+    Tool_Manifest_File: $(System.DefaultWorkingDirectory)/.config/dotnet-tools.json
+    Coverage_Reports_Glob: $(Build.SourcesDirectory)/**/coverage.cobertura.xml
+    Coverage_Report_Target_Directory: $(Build.SourcesDirectory)/TestResults/coveragereport
+    Coverage_Report_Target_File: $(Build.SourcesDirectory)/TestResults/coveragereport/Cobertura.xml
+
+  steps:
+  - checkout: self
+    displayName: Checkout
+    fetchDepth: 1
+  - task: UseDotNet@2
+    displayName: Setup .NET SDK
+    inputs:
+      packageType: 'sdk'
+      useGlobalJson: true
+      performMultiLevelLookup: true
+  - task: DotNetCoreCLI@2
+    displayName: Install .NET Tools
+    inputs:
+      command: 'custom'
+      custom: 'tool'
+      arguments: 'restore --configfile $(NuGet_Configuration_File) --tool-manifest $(Tool_Manifest_File) --no-cache'
```

Comment:
I'm not sure if this bothers me but by having a single tool-manifest we restore all dotnet tools we use. Right now this leads to `stryker` being restored although it's not used in the code-coverage job. A shame that there seems to be no way of handling this better

Reply:
Just as a note for later discussions:
What about having multiple dotnet-tools.json with the "isRoot" set to false?
Can this be a way to achieve this?

Reply:
Separate `tool-manifest` files seems non-tool-manifesty.
Lest's completely parameterize all templates, including the respective .NET tool version (latest if not specified), and move those job-tempates to a separate repo, so everyone could resuse them.